jest.config.ts

```diff
@@ -24,4 +24,5 @@ export default {
   testMatch: ["**/tests/**/*.test.ts"],
   preset: "ts-jest",
   testEnvironment: "node",
+  setupFiles: ["<rootDir>/setEnvVars.js"],
```

Comment:
Question: `<rootDir>` is this a keyword or something that is automatically evaluated?

Reply:
Yeah it's a jest config thing - it's in the docs https://jestjs.io/docs/configuration#setupfilesafterenv-array